__tests__/components/EmailInput.test.tsx

```diff
@@ -13,40 +13,45 @@ describe.skip("Email Input", () => {
     onChange,
     testId: "email-input",
     placeholder: "your@mail.com",
-    helpertext: "Enter your email"
+    helperText: "Enter your email",
   };
 
   it('Should render with helper text', () => {
     render(<EmailInput {...defaultEmailInputProps} />);
 
     const emailComponent = screen.getByTestId(defaultEmailInputProps.testId!);
     expect(emailComponent).toBeInTheDocument();
-
-    expect(emailComponent).not.toHaveAttribute('required');
-    expect(emailComponent).toHaveAccessibleName('Email');
-
-    expect(within(emailComponent).getByText(defaultEmailInputProps.helpertext!)).toBeInTheDocument();
-    expect(emailComponent).toHaveAccessibleName(defaultEmailInputProps.label);
+    const inputElement = screen.getByRole('textbox');
+    expect(inputElement).not.toHaveAttribute('required');
+    expect(inputElement).toHaveAccessibleName(defaultEmailInputProps.label);
+    expect(within(emailComponent).queryByText(defaultEmailInputProps.helperText!)).toBeInTheDocument();
     const placeholder = screen.queryByPlaceholderText(defaultEmailInputProps.placeholder!);
     expect(placeholder).toBeInTheDocument();
   });
 
   it('Should render with error text, hiding helper text', () => {
-    render(<EmailInput {...defaultEmailInputProps} error="Some error happen" />);
 
+    let defaultEmailInputProps: EmailInputProps = {
```

Comment:
Do we really need to redefine `defaultEmailInputProps` again?

Reply:
No, we don't! Deleted unnecessary repetitions